xml/Currency.xml

```diff
@@ -272,6 +275,9 @@
             <name>
                 <narrative>Kroon</narrative>
             </name>
+            <description>
+                <narrative>Removed from ISO Currency codelist</narrative>
```

Comment:
I think in this case, it would be preferable to mark the code as `status="withdrawn"`. There’s an optional `withdrawn-date` too, i.e. https://github.com/IATI/IATI-Codelists/blob/2e4d110dff05e03e82de90f892ceae28afe31462/codelist.xsd#L57